ebean-test/src/test/java/org/tests/cache/TestBeanCache.java

```diff
@@ -157,6 +158,80 @@ public void idsInExpression() {
     }
   }
 
+  @Test
+  public void testFindSet() {
```

Comment:
This is the same test as 'idsInExpression' maybe refactor, so that we can pass a `Colletion finder(Query)` lambda expression?